src/ExpressionEvaluator/Core/Source/ExpressionCompiler/PDB/PdbHelpers.cs

```diff
@@ -204,29 +204,28 @@ public static ConstantValue GetSymConstantValue(ITypeSymbol type, object symValu
                         return ConstantValue.Create(string.Empty);
                     }
 
-                    var str = symValue as string;
-                    if (str == null)
+                    if (!(symValue is string str))
                     {
                         return ConstantValue.Bad;
                     }
 
                     return ConstantValue.Create(str);
 
                 case SpecialType.System_Object:
-                    if (symValue is int && (int)symValue == 0)
+                    if (symValue is int intVal2 && intVal2 == 0)
```

Comment:
`symValue is 0`

Reply:
That would box the 0 though.

Reply:
not on master. this case has been optimised and would generate the exact same code as above.

Reply:
`is 0` is not (no pun intended) idiomatic for the roslyn codebase.

Reply:
You have to start somewhere.  Embrace it.

Reply:
sorry, i misread, i thought this was about `intVal2 is 0` instead of `isVal2 == 0`.  I didn't realize the suggestion was on `symValue`.  Will update.